plugins.rst

```diff
@@ -24,14 +24,15 @@ The following is the list of official plugins by |omv|. Their availability depen
 * **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
 * **Diskstats**: Complementary plugin to extend system statistics collection by adding I/O statistic graphs.
 * **Forked-daap**: Provides a daap protocol music server.
-* **FTP**: Provides a modular FTP/SFTP/FTPS server.
+* **FTP**: Provides a modular FTP/FTPS server.
 * **LVM2**: LVM managing. Create volume groups and logical partitions.
 * **NUT**: Controlling and configuring UPS. The driver support is based on NUT.
 * **Onedrive** (amd64, arm64, armhf, i386 only): Synchronizing a shared folder with Microsoft OneDrive cloud storage.
 * **Podman**: A tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers. Gives users the ability to pull the latest images of the containers that are used by some of the plugins
 * **Shairport**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.
 * **ShareRootFs**: Provides shared directories on root file system.
 * **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks.
+* **SFTP**: Provides an SFTP server (on another port than SSH).
```

Comment:
This is a 3rd party plugin provided by omv-extras.org which should not be listed here. All the plugins listed here are maintained by the core OMV project.